src/smart-components/Channel/context/utils.js

```diff
@@ -8,19 +8,19 @@ import { OutgoingMessageStates } from '../../../utils/exports/getOutgoingMessage
 const UNDEFINED = 'undefined';
 const { SUCCEEDED, FAILED, PENDING } = getSendingMessageStatus();
 
-export const scrollIntoLast = (intialTry = 0) => {
+export const scrollIntoLast = (intialTry = 0, scrollRef) => {
```

Comment:
@HoonBaek  use this pattern in threading

Reply:
Looks good, thank you 🙏

Reply:
variable name should be `initialTry`?

Reply:
si... Will apply before merging